css/at-rules/document.json

```diff
@@ -38,14 +38,8 @@
               {
                 "prefix": "-moz-",
                 "version_added": "61",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.moz-document.content.enabled",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "Disabled by default in web pages, except for an empty <code>url-prefix()</code> value, which is supported due to its <a href='https://css-tricks.com/snippets/css/css-hacks-targeting-firefox/'>use in Firefox browser detection</a>. Still supported in user stylesheets."
+                "partial_implementation": true,
+                "notes": "Only supports an empty <code>url-prefix()</code> value, which is supported due to its <a href='https://css-tricks.com/snippets/css/css-hacks-targeting-firefox/'>use in Firefox browser detection</a>. Still supported in user stylesheets."
```

Comment:
Are you sure this quirk isn't also on Firefox Android?

Reply:
These are the notes for Firefox Android actually!

Reply:
Oh right, I saw the next chunk in the diff and thought this was desktop.

Looking at the full diff I have another ask: we should capture that there's partial support in desktop too with the same statement and note. I think that it should be possible to remove all flag data from BCD and the result should be what's supported in default configs, but that isn't true if support is hidden in notes of flag entries.